elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -0,0 +1,189 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: navigation;
+
+:host {
+  --pfe-navigation__overlay--BackgroundColor: var(--pfe-theme--overlay--BackgroundColor, rgba(37, 37, 37, 0.5));
+
+  // Default spacing values
+  --pfe-navigation--Padding--vertical: #{pfe-var(container-padding)};
+  --pfe-navigation--Padding--horizontal: #{pfe-var(container-padding)};
+
+  --pfe-navigation--Padding: 0 #{pfe-local(Padding--horizontal)};
+
+  // Darkest is the default background color
+  --pfe-navigation--BackgroundColor: #{pfe-color(surface--darkest)};
+  --pfe-navigation--Color: #{pfe-color(surface--darkest--text)};
+  --pfe-navigation--Border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} transparent;
+  --pfe-navigation--MinHeight: 72px;
+
+  // Reduce padding for print
+  @media print {
+    --pfe-navigation--Padding: calc(#{pfe-local(Padding--vertical)} / 2) #{pfe-local(Padding--horizontal)};
+  }
+
+  @media screen and (min-width: pfe-breakpoint(sm)) {
+    --pfe-navigation--Padding--horizontal: calc(#{pfe-var(container-padding)} * 2);
+  }
+
+  display: block;
+  z-index: 2;
+  width: 100%;
+}
+
+:host(.sticky) {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  z-index: 99;
+}
+
+// Note: These styles do not pick up in Edge
+::slotted([slot="logo"]) {
+  margin: 0 !important;
+  max-height: #{pfe-local(MinHeight)};
+}
+
+::slotted([slot="mobile-login"]),
+::slotted([slot="mobile-language"]) {
+  color: var(--pfe-broadcasted--color--ui-link, #{pfe-color(ui-link)});
+  text-decoration: var(--pfe-broadcasted--link--text-decoration, #{pfe-var(link--text-decoration)});
+}
+
+@each $icon in (user, globe) {
+  ::slotted([icon="#{$icon}"]) {
+    --pfe-navigation__icon--Background: #{map-get($icons, #{$icon})} no-repeat;
+  }
+}
+
+.pfe-navigation {
+  &__wrapper {
+    position: relative;
+    z-index: 2;
```

Comment:
Same question here about setting a variable with a fallback.

Reply:
Good idea, I've updated these and added a z-index map